basic-application/basic-application-webapp/src/main/java/fr/openwide/core/basicapp/web/application/BasicApplicationApplication.java

```diff
@@ -105,7 +110,9 @@ protected IConverterLocator newConverterLocator() {
 		converterLocator.set(TechnicalUser.class, UserRenderer.get());
 		converterLocator.set(BasicUser.class, UserRenderer.get());
 		converterLocator.set(UserGroup.class, UserGroupRenderer.get());
+		converterLocator.set(GenericLocalizedGenericListItem.class, LocalizedGenericListItemRenderer.get());
```

Comment:
Maybe we should open the discussion with the Wicket team to support inheritance?